keunhokim/users/views.py

```diff
@@ -0,0 +1,90 @@
+import json
+import bcrypt
+import jwt
+
+from django.http            import JsonResponse,HttpResponse
+from django.core.exceptions import ValidationError
+from utils                  import validate_email,validate_password
+
+from django.views import View
+from .models      import User,Coupon
+from my_settings  import SECRET_KEY, algorithm
+
+class UserSignUpView(View):
+    def post(self,request):
+        data = json.loads(request.body)
+
+        try:
+            name         = data.get('name',None)
+            email        = data.get('email',None)
+            phone_number = data.get('phone_number',None)
+            image_url    = data.get('image_url', None)
+            password     = data.get('password', None)
+            last_name    = data.get('last_name',None)
+            first_name   = data.get('first_name',None)
```

Comment:
```suggestion
            image_url    = data.get('image_url', None)
            last_name    = data.get('last_name',None)
            first_name   = data.get('first_name',None)
```
이 값들은 회원가입 필수 값이 아니지 않나요?

Reply:
필요없는 data 삭제했습니다